### DISCUSSION TITLE
Error loading tags: Java.lang.StackOverFlowError

### DISCUSSION THREAD

**Discussion by azdave2020** [nextgenhealthcare/connect#5304] [Q&A]
This error started last night and i cant see how to repair it.
Mirth Connect Server 3.10.1

Built on January 6, 2021

Server ID: e05420cc-68c1-4246-86fd-e0e17c5e16c5

Java version: 1.8.0_331

Running on windows server 2019 32GB ram
What are my next steps?

![image](https://user-images.githubusercontent.com/90641786/180455196-101cd720-969c-4a7d-be47-51d8e298101f.png)

**Comment by jonbartels** [comment]
Is the error shown from the MC server or from the MC client? Is this error shown only in the GUI or also in `mirth.log`.

Is MC still running despite the error being shown? Eg is the process running, ports open, etc?

If the client - Use the MC Admin Launcher and not raw `javaws`. MCAL solves a lot of caching pain points for you.

If the server - Did this start after a server restart or other maintenance? (like https://github.com/nextgenhealthcare/connect/discussions/5306) Did this happen on a live server?
